### ISSUE TITLE
-resilient documentation is vague in ojph_expand

### ISSUE THREAD

**Issue by palemieux** [aous72/OpenJPH#148]
The documentation should clarify that `-resilient` takes either `true` or `false` as permitted values.

```
" -resilient <true | false>  if `true`, the decoder will not exit when running into recoverable errors\n"
    "                                    in the codestream. Default: `false`\n\n"
```

**Comment by aous72** [comment]
Hi Pierre,

Sorry for the confusion.  I agree it might have been confusing. I will fix it soon.

Kind regards,
Aous.

**Comment by aous72** [comment]
This issue was addressed in release 0.15.0.
If there are any issues, please reopen this issue.
